pkg/dbt_cloud/paginate.go

```diff
@@ -90,7 +95,23 @@ func (c *Client) GetData(url string) []any {
 		allResponses = append(allResponses, response.Data...)
 	}
 
-	return allResponses
+	return allResponses, nil
+}
+
+func (c *Client) GetData(url string) []any {
+	rawData, err := c.GetRawData(url)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	allData := make([]any, len(rawData))
+	for i, data := range rawData {
+		err := json.Unmarshal(data, &allData[i])
```

Comment:
The acceptance test is now passing, which is great 👍 , but I am just wondering why we need to add a new unmarshalling step to the process

Reply:
Small performance tweak. We added that to try and prevent the data from needing to be marshalled and unmarshaled again by the caller (ie: 161-163 or 180-182). Might make sense to try and swap some of the other client calls over to the GetRawResponse method.

Reply:
@b-per The magic lies in Go's `json.RawMessage` type, so the actual data doesn't get unmarshalled just to read the pagination info.